### PR TITLE
Perspective blob + Blinky blob

### PR DESCRIPTION
**Pull request by binarysudoku**
## About the PR 

- Resprites blob to be in perspective and adds more jiggly animation + blinking
- Slightly increases alpha of full-health blob for legibility

blob_test_4

## Why's this needed? 

New blob looks cuter and blinks so it looks nicer

## Changelog

```
(u)Erinexx:
(+)New, jigglier blob sprites.
```

### CODE REVIEW

**Comment by binarysudoku** [comment]
NOTE: probably gonna pixel shift the blob nucleus up a bit for visual improvements

File: code/obj/blob.dm [hunk 1]
```diff
@@ -1176,10 +1193,11 @@ var/image/blob_icon_cache
 			added = 0
 
 
-/obj/blob/wall
+/obj/blob/wall // TODO: Add check for if theres a wall to the south, if so replace sprite to pseudo-tile
 	name = "thick membrane"
 	desc = "This blob is encased in a tough membrane. It'll be harder to get rid of."
-	state_overlay = "new_wall"
+	//icon_state = null
```
Comment (ZeWaka) [inline code/obj/blob.dm:1199]: ?

File: code/obj/blob.dm [hunk 2]
```diff
@@ -1176,10 +1193,11 @@ var/image/blob_icon_cache
 			added = 0
 
 
-/obj/blob/wall
+/obj/blob/wall // TODO: Add check for if theres a wall to the south, if so replace sprite to pseudo-tile
```
Comment (ZeWaka) [inline code/obj/blob.dm:1196]: TODO?

File: code/obj/blob.dm [hunk 3]
```diff
@@ -968,7 +985,7 @@ var/image/blob_icon_cache
 /datum/projectile/slime
 	name = "slime"
 	icon = 'icons/obj/projectiles.dmi'
-	//state_overlay = "new_slime"
+	//state_overlay = "slime"
```
Comment (ZeWaka) [inline code/obj/blob.dm:988]: commented out?

File: code/obj/blob.dm [hunk 4]
```diff
@@ -571,6 +587,7 @@ var/image/blob_icon_cache
 	poison_coefficient = 0.5
 	poison_depletion = 3
 	var/nextAttackMsg = 0
+	var/blink_overlay
```
Comment (ZeWaka) [inline code/obj/blob.dm:590]: unused?

File: code/obj/blob.dm (hunk 2 above)
Comment (binarysudoku) [reply code/obj/blob.dm:1196]: oh shit sorry this is left over, ill remove it

File: code/obj/blob.dm (hunk 3 above)
Comment (binarysudoku) [reply code/obj/blob.dm:988]: i did a batch-rename of everything that was named "new_thing" for consistency

Comment (binarysudoku) [reply code/obj/blob.dm:987]: ok its commented out in main so i stand by this change

**Pull request merged**